cmd/create/operatorroles/by_clusterkey.go

```diff
@@ -331,6 +340,9 @@ func buildCommands(r *rosa.Runtime, env string,
 		if managedPolicies {
 			iamTags[tags.ManagedPolicies] = helper.True
 		}
+		if hostedCPPolicies {
+			iamTags[tags.HypershiftPolicies] = helper.True
```

Comment:
Just noticed this now, this should be HcpPolicies or similar, as this tag is something the user can be exposed to.

Reply:
Are you referring to the tag's name in AWS?
I can change it, need to check with BU what is the preferred name.
Thanks

Reply:
Yes the tag name in AWS, just making sure whatever the customer's can see are all hcp instead of hs